client/influxdb_internal_test.go

```diff
@@ -0,0 +1,84 @@
+package client
+
+import (
+	"encoding/json"
+	"io"
+	"net/http"
+	"net/url"
+	"strconv"
+	"strings"
+	"testing"
+)
+
+func TestCreateHTTPRequest(t *testing.T) {
+	c := &Client{}
+	c.url = url.URL{Scheme: "https", Host: "127.0.0.1"}
+	c.userAgent = "UserAgent"
+
+	q := Query{Command: "CommandToRun", Database: "DatabaseName", Chunked: true}
+
+	req, err := c.createHTTPRequest(q)
+	if err != nil {
+		t.Fatalf("unexpected error.  expected %v, actual %v", nil, err)
+	}
+
+	if req.URL.String() != "https://127.0.0.1/query?chunked=true&db=DatabaseName&q=CommandToRun" {
+		t.Fatalf("error request URL not being constructed correctly")
+	}
+	if req.Header["User-Agent"][0] != "UserAgent" {
+		t.Fatalf("erro User Agent header not being set correctly.  expected %v, actual %v", "UserAgent", req.Header["User-Agent"][0])
+	}
+}
+
+func TestProcessResponse(t *testing.T) {
+
+	bod := `{"results":[{"statement_id":0,"series":[{"name":"average_temperature","columns":["time","degrees","location"],"values":[[1439856000000000000,82,"coyote_creek"]]}]}]}`
+
+	status := 200
+
+	NewRespBodyFromString := func(body string) io.ReadCloser {
+		return &dummyReadCloser{strings.NewReader(body)}
+	}
+
+	resp :=
```

Comment:
This looks like bad formatting. Did you run this through `go fmt`?

Reply:
This tests are not necessary after previous changes.